### ISSUE TITLE
[Dropdown] Values are parsed incorrectly when they contain <, ≤, etc

### ISSUE THREAD

**Issue by GammaGames** [fomantic/Fomantic-UI#1207]
# Bug Report

## Steps to reproduce
1. Create a dropdown
2. Set its values to `<`, `>`, `≤`, and `≥`
3. Get values and inspect the result

## Expected result
Get the values for `<` and company

## Actual result
Get `&;amp;amp;amp;lt;` and company. The number of `&amp;` might be related to number of selected values?

## Testcase
https://jsfiddle.net/4cqtnf0d/

## Screenshot (when possible)
![image](https://user-images.githubusercontent.com/7832163/70192293-889cea80-16b8-11ea-8f1b-b029ed58c46f.png)

## Version
2.8.2

**Comment by GammaGames** [comment]
If anyone needs a workaround to fix this, I made a little function that will decode the values:

```javascript

let decodeEntity = (text, iterations) => {
  iterations = iterations || 10;
  for(let i = 0; i < iterations; i++) {
    text = $("<textarea></textarea>").html(text).text();
  }
  return text;
}
```

The second argument is the number of elements. It appears that if you have many elements selected and remove some of them, you still have to decode to the length of the longest count of options. To fix this, you can pass in the total length of options like so:

```javascript
let option_count = $(".ui.dropdown div[data-value], .ui.dropdown a[data-value]").length;
let values = $('.ui.dropdown').dropdown('get values').map(val => decodeEntity(val, option_count));
console.log(values);
```

[jsfiddle example](https://jsfiddle.net/z2jhns5a/)

**Comment by GammaGames** [comment]
I think this is also messing with the changing logic, when a value is removed it can't find any match in the values for `1<` so it doesn't remove the value in the array for `1&amp;amp;amp;amp;amp;amp;amp;amp;lt;`. You can see that by unselecting elements in the dropdown in [this fiddle](https://jsfiddle.net/x5k286j0/).

![image](https://user-images.githubusercontent.com/7832163/70257883-6f437f00-1748-11ea-8755-a8e4fda9bbb5.png)

This also highlights that the elements are further encoded, so I will modify the above decoding function to more accurately decode.

**Comment by lubber-de** [comment]
Fixed by #1210 . See your adjusted jsfiddle here https://jsfiddle.net/kjqoa89e/

**Comment by lubber-de** [comment]
@GammaGames I also fixed the remove label issue now
See your adjusted jsfiddle here  https://jsfiddle.net/vckzmxaf/
